test/Microsoft.Dnx.Testing.CommonUtils/DnxFunctionalTestBase.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class DnxSdkFunctionalTestBase
+    {
+        public const string SdkVersionForTestingEnvName = "DNX_SDK_VERSION_FOR_TESTING";
+
+        static DnxSdkFunctionalTestBase()
+        {
```

Comment:
You can use `IUseFixture<T>` http://stackoverflow.com/questions/12976319/xunit-global-setup-teardown